templates/navBar.php

```diff
@@ -108,8 +108,9 @@
 </style>
 
 <nav>
-  <div class="logo">
-    <p>Kidney Exchange</p>
+  <div class="logo" style="padding:0px;">
+    <img src="./images/logo3.png" width="100px" hight="100px" style="float:left;">
+    <p style="float:left;margin-top:30px;margin-left:20px;">Kidney Exchange</p>
```

Comment:
Can you add this style using CSS inside the `style` tag present on top of this file?
Generally, it is better to avoid adding styles within the HTML tag.